lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -1943,10 +1957,22 @@ bool TFFunctionPartition::markFunction() {
   // Not that we know all of the instructions we'll be moving over, find the end
   // point by finding the nearest common post dominating ancestor of the marked
   // instructions.
+  //
+  // As we walk through all markings, if there's any Move/Copy while the
```

Comment:
Move/Copy ==> Copy/Send

I think Move is fine?

Reply:
Good catch. Done.